Makefile

```diff
@@ -67,7 +67,7 @@ endif
 TFTPDIR ?= /var/lib/tftpboot
 
 # Warewulf directory paths
-VARLIST += WWCLIENTDIR WWCONFIGDIR WWPROVISIONDIR WWOVERLAYDIR WWCHROOTDIR WWTFTPDIR WWDOCDIR WWDATADIR
+VARLIST += WWCLIENTDIR WWCONFIGDIR WWPROVISIONDIR WWOVERLAYDIR WWCHROOTDIR WWTFTPDIR WWDOCDIR WWDATADIR TMPDIR
```

Comment:
I think `TMPDIR` should be renamed so it isn't confused with the canonical environment variable and what is returned by `os.TempDir`.